documentation/concepts.md

```diff
@@ -152,6 +196,11 @@ be aware. Note that this might be an area for future improvement to increase
 usability. However, this is akin to other similar systems, like Kafka, where
 you must first create a topic and then you publish to that topic.
 
+
+```
+The typical use case for a producer not caring if the ACK is returned or not is an IoT device or a sensor. This means that for the sensor it is not important to know if Liftbridge indeed got to record the event. For a more regulated system, one could "assume" acknowledgements are important to the producer since the recorded truth now resides within Liftbridge, ref event sourcing.
+```
```

Comment:
```suggestion
> **Use Case Note**
>
> The typical use case for a producer not caring if the ack is returned or not is an IoT device or a sensor. This means that for the sensor, it is not important to know if Liftbridge indeed got to record the event. For a more regulated system, one could assume acknowledgements are important to the producer since the recorded truth now resides within Liftbridge, as is the case in an event-sourced system.
```